src/main/java/de/qabel/desktop/ui/accounting/identitycontextmenu/IdentityContextMenuController.java

```diff
@@ -93,6 +99,20 @@ public void openQRCode() {
         Platform.runLater(() -> qrcodeController.showPopup());
     }
 
+    public void showIdentityEdit() {
+        createIdentityEdit();
+        Platform.runLater(() -> identityEditController.show());
+    }
+
+    IdentityEditView createIdentityEdit() {
+        if (identityEditView == null) {
+            identityEditView = new IdentityEditView(generateInjection("identity", identity));
+            identityEditController = (IdentityEditController) identityEditView.getPresenter();
+        }
+        return identityEditView;
+    }
+
+    @Deprecated
```

Comment:
it will be removed / replaced in process so the deprecation is currently just for me